src/UI/Components/ExpertBanner/ExpertBanner.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from "react";
+import { Banner } from "@patternfly/react-core";
+
+interface CustomEvent extends Event {
```

Comment:
Isn't CustomEvent already a type that is available ?

Reply:
It is, but without this extension I got typing errors when trying to use that detail parameter which should be available.